data/json/items/resources/metal.json

```diff
@@ -6,13 +6,14 @@
     "description": "A steel pipe, makes a good melee weapon.  Useful in a few crafting recipes.",
```

Comment:
```suggestion
    "description": "A steel pipe; makes a good melee weapon.  Useful in a few crafting recipes.",
```

Reply:
not sure why we need to change things that weren't changed from me in the first place. But ok, I am no native English speaker, I trust you on that one.